addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -718,6 +726,14 @@ var FieldInteger = InputField.extend({
     isSet: function () {
         return this.value === 0 || this._super.apply(this, arguments);
     },
+    _renderEdit: function () {
+        this._super.apply(this, arguments);
+        if( this.$input ){
```

Comment:
Why? Is there *any* situation where InputField would not have created the $input and we'd still expect the field to work correctly?